deploy/docker/deploy-internalstage.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Just so I'm clear the purpose of this is to push to the staging virtual machine right?